factories/expense.py

```diff
@@ -0,0 +1,17 @@
+import random
+import factory
+
+from house.models import House
+from expenses.models import Expenses
+
+
+class ExpenseFactory(factory.Factory):
```

Comment:
I have found something that maybe will work better for us, try to inherit from DjangoModelFactory